tables/afr-za-g2.ctb

```diff
@@ -0,0 +1,850 @@
+# liblouis: Afrikaans Braille Code (Grade 2)
+#
+# -----------
+#-name: Afrikaans braille, grade 2
+#-display-name: Afrikaans (UBC) contracted braille
+#
+#+locale:af
+#+type:literary
+#+contraction:full
+#+grade:2
+#+system:ueb
```

Comment:
I assume this is an oversight.

Reply:
Or maybe not... But I'm not sure that the system *is* actually UEB or based on UEB?

Reply:
I just noticed that the grade 1 table just does a `include en-ueb-g1.ctb`. So for that table, `#+system:ueb` should be added. Not sure about the grade 2 table.

Reply:
In the display-name, "UBC" is mentioned. Looked it up: https://sabrailleauthority.org.za/unified-braille-code-ubc.

Reply:
I think I will change this into `#+system:ubc-af` and add a bit of explanation. I'll add this link to the page on UBC, and possibly I could mention that "ueb" can be considered to be the same as "ubc-en".